Kudu.Core.Test/Functions/FunctionManagerTests.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System.Collections.Generic;
+using Kudu.Core.Functions;
+using Kudu.Core.Tracing;
+using Newtonsoft.Json.Linq;
+using Xunit;
+
+namespace Kudu.Core.Test.Functions
+{
+    public class FunctionManagerTests
```

Comment:
There were no unit tests for any of this code, so I'm adding some here. We really need to make sure we're adding tests with code.